### ISSUE TITLE
Breadcrumb: High contrast, contrast ratio below 4.5

### ISSUE THREAD

**Issue by woofton** [microsoft/fluentui#20137]
### Environment Information

- **Package version(s)**: Latest
- **Browser and OS versions**: Edge, Windows 10

### Describe the issue:

In windows, turn on high contrast. Then in the Edge browser, go to https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb, run accessibility fast pass. You will notice that the last item in the bread crumb is below the contrast ratio of 4.5

### Please provide a reproduction of the issue in a codepen:

In windows, turn on high contrast. Then in the Edge browser, go to https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb, run accessibility fast pass. You will notice that the last item in the bread crumb is below the contrast ratio of 4.5

#### Actual behavior:

The last item in the bread crumb is below the contrast ratio of 4.5

#### Expected behavior:

The last item in the bread crumb should be at or above the contrast ratio of 4.5

### Documentation describing expected behavior

The last item in the bread crumb should be at or above the contrast ratio of 4.5

**Comment by woofton** [comment]
@bsunderhus any update?
